core/shared/src/main/scala/reftree/geometry/Point.scala

```diff
@@ -122,10 +134,15 @@ object Rectangle {
   def union(rectangles: Seq[Rectangle]) = rectangles.reduce(_ union _)
 
   /** Parse an SVG rectangle */
-  def fromString(string: String) = {
-    val Polyline(Seq(topLeft, widthHeight)) = Polyline.fromString(string)
-    Rectangle(topLeft, topLeft + widthHeight)
-  }
+  def fromString(string: String) =
+    Polyline.fromString(string) match {
+      case Polyline(Seq(topLeft, widthHeight, _@_*)) =>
+        Rectangle(topLeft, topLeft + widthHeight)
+      case Polyline(Seq(topLeft)) =>
+        Rectangle(topLeft, topLeft)
```

Comment:
Is this valid SVG?

Reply:
Again, not confident about that one, but throwing unexpected exceptions is not something fine from my feelings.